### PR TITLE
refactor: simplify appLayout and remove topbar variant logic

### PR DESCRIPTION
**Pull request by J1za**
## Linear Ticket
https://linear.app/autokitteh/issue/UI-930/topbar-join-all-states-and-display-on-all-screens
## What type of PR is this? (check all applicable)

-   [ ] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/app.tsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ export const App = () => {
 	return (
 		<BrowserRouter>
 			<AKRoutes>
-				<Route element={<AppLayout />} path="/">
+				<Route element={<AppLayout topbarHidden={true} />} path="/">
```
Comment (RonenMars) [inline src/app.tsx:67]: I think that it should be enough:
```suggestion
				<Route element={<AppLayout topbarHidden />} path="/">
```

Comment (RonenMars) [reply src/app.tsx:67]: Also please rename the property to: `hideTopbar`

File: src/components/templates/appLayout.tsx [hunk 2]
```diff
@@ -52,7 +51,7 @@ export const AppLayout = ({ className, topbarVariant }: { className?: string; to
 			<Sidebar />
 
 			<div className="mb-2 flex flex-1 flex-col">
-				{topbarVariant ? <ProjectConfigTopbar variant={topbarVariant} /> : null}
+				{!hideTopbar ? <ProjectConfigTopbar /> : null}
```
Comment (RonenMars) [inline src/components/templates/appLayout.tsx:54]: Let's make the condition positive (without `!`):
```suggestion
				{hideTopbar ? null : <ProjectConfigTopbar />}
```

Comment (J1za) [reply src/components/templates/appLayout.tsx]: changed

**Pull request merged**